app/views/admin/infobase/_orders.html.haml

```diff
@@ -0,0 +1,17 @@
+.ui.top.attached.tabular.menu
+  %a.item{"data-tab" => "Monday"} Monday
```

Comment:
` %a.item{ data: { tab: week_day }}`
that's first of all. Secondly this really looks like a place for a loop, wtf man :)